CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Unreleased
 
+# 9.11.0 (Patch release)
```

Comment:
```suggestion
# 9.11.0 (Feature release)
```
As we'll be releasing a feature release of GOVUK Frontend and we're changing the minor version in the version number (i.e: that middle number), this is a feature release rather than a patch release.

Reply:
@vanitabarrett Ah, I see...Thanks! :)